src/binding.cc

```diff
@@ -142,24 +142,26 @@ void ConstantsInit(v8::Local<v8::Object> exports) {
   NODE_DEFINE_CONSTANT(topicConstants, RdKafka::Topic::OFFSET_STORED);
   NODE_DEFINE_CONSTANT(topicConstants, RdKafka::Topic::OFFSET_INVALID);
 
-  exports->Set(Nan::New("topic").ToLocalChecked(), topicConstants);
+  Nan::Set(exports, Nan::New("topic").ToLocalChecked(), topicConstants);
 
-  exports->Set(Nan::New("err2str").ToLocalChecked(),
+  Nan::Set(exports, Nan::New("err2str").ToLocalChecked(),
     Nan::GetFunction(Nan::New<v8::FunctionTemplate>(NodeRdKafkaErr2Str)).ToLocalChecked());  // NOLINT
 
-  exports->Set(Nan::New("features").ToLocalChecked(),
+  Nan::Set(exports, Nan::New("features").ToLocalChecked(),
     Nan::GetFunction(Nan::New<v8::FunctionTemplate>(NodeRdKafkaBuildInFeatures)).ToLocalChecked());  // NOLINT
 }
 
-void Init(v8::Local<v8::Object> exports, v8::Local<v8::Object> module) {
-  AtExit(RdKafkaCleanup);
+void Init(v8::Local<v8::Object> exports, v8::Local<v8::Value> m_, void* v_) {
+  v8::Local<v8::Context> context = Nan::GetCurrentContext();
+  node::Environment* env = node::GetCurrentEnvironment(context);
+  AtExit(env, RdKafkaCleanup, NULL);
```

Comment:
This seems to be reason why it doesn't work for node <= 8
This may do the trick:
```
  #if NODE_MAJOR_VERSION <= 8
  AtExit(RdKafkaCleanup);
  #else
  AtExit(node::GetCurrentEnvironment(Nan::GetCurrentContext()), RdKafkaCleanup, NULL);
  #endif
```